ivadomed/preprocessing.py

```diff
@@ -0,0 +1,38 @@
+# Transformation to perform on image before network processing
+import nibabel as nib
+import numpy as np
+import ivadomed.utils as imed_utils
+
+
+def get_midslice_average(path_im, ind, slice_axis=0):
+    """
+    Extract an average 2D slice out of a 3D volume. This image is generated by
+    averaging the 7 slices in the middle of the volume
+    Args:
+        path_im (string): path to image
+        ind (int): index of the slice around which we will average
+        slice_axis (int): Slice axis according to RAS convention
```

Comment:
does it matter that it is according to the RAS convention? here, it only refers to a dimension, right?

Reply:
addressed in https://github.com/ivadomed/ivadomed/pull/271#pullrequestreview-436913166